ms-engine-spring-jaxrs/src/main/java/net/trajano/ms/engine/jaxrs/JaxRsPath.java

```diff
@@ -74,6 +77,7 @@ public void apply(final Router router,
         final Handler<RoutingContext> jaxRsHandler,
         final Handler<RoutingContext> failureHandler) {
 
+        System.out.println(this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)